Jenkinsfile

```diff
@@ -69,10 +72,10 @@ fabric8UITemplate{
 
 // deploy a snapshot fabric8-ui pod and notify pull request of details
 if (ciDeploy){
-    def prj = 'fabric8-ui-'+ env.BRANCH_NAME
-    prj = prj.toLowerCase()
-    def route
-    timeout(time: 10, unit: 'MINUTES') {
+   timeout(time: 50, unit: 'MINUTES') {
```

Comment:
50 minutes sounds pretty long.

Reply:
yeah I understand but sometimes even 1h is less,
we face errors like below where Jenkins-slave is unavailable for the random amount of time.
```
Still waiting to schedule task
Jenkins doesn’t have label dockerfabric8_planner_27
```
In best case 10m is sufficient but current builds may take more time and timeout with a small timeout value.
`Cancelling nested steps due to timeout. Could not connect to jenkins-slave-prxp7-mp5d9 to send interrupt signal to process`